src/components/vue-form-renderer.vue

```diff
@@ -205,42 +207,45 @@ export default {
         this.$emit('submit', this.transientData);
       }
     },
-    validateElements(elements) {
-      elements.forEach(element => {
-        if (element.validator && element.validator.errorCount !== 0) {
-          this.valid = false;
-          this.errors.push(element.validator.errors.errors);
+    getDataAndRules(items) {
+      let data = {}, rules = {};
```

Comment:
You've taken the arguments and moved them into the method, but you should also be thinking about what the method is actually doing with them. You've replaced one problem with another: Instead of recursively passing down props by reference, you're now holding onto three different versions of the props that you spread into each other.